Configuration.props

```diff
@@ -113,4 +113,7 @@
     <AndroidSupportedTargetAotAbisForConditionalChecks Condition=" !$(AndroidSupportedTargetAotAbisForConditionalChecks.StartsWith (':')) " >:$(AndroidSupportedTargetAotAbisForConditionalChecks)</AndroidSupportedTargetAotAbisForConditionalChecks>
     <AndroidSupportedTargetAotAbisSplit>$(AndroidSupportedTargetAotAbis.Split(':'))</AndroidSupportedTargetAotAbisSplit>
   </PropertyGroup>
+  <PropertyGroup>
+    <RemapAssemblyRefTool>$(ManagedRuntime) $(MSBuildThisFileDirectory)bin/Build$(Configuration)/remap-assembly-ref.exe</RemapAssemblyRefTool>
```

Comment:
This should use `\` instead of `/` for paths -- xbuild/msbuild should properly replace, as appropriate (but test! sometimes xbuild is buggy), and this should quote paths which may contain spaces, which is the case for `$(MSBuildThisFileDirectory)`:

    <RemapAssemblyRefTool>$(ManagedRuntime) "$(MSBuildThisFileDirectory)bin\Build$(Configuration)\remap-assembly-ref.exe"</RemapAssemblyRefTool>